docs/wiki/Development-and-Contributing.md

```diff
@@ -10,6 +10,120 @@ Before beginning work, chat with us in the
 The pace of development is pretty fast right now, so you'll be expected to keep
 your pull requests moving through the review process.
 
+### Prerequisites for Bison Wallet Development
```

Comment:
Oh, I see, this is new... But we have https://github.com/decred/dcrdex/wiki/Download-and-Install#building-from-source cant we just point there? We don't need the same info in two places do we?